my-sms/lib/tasks/migrate_student_title.rake

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+namespace :db do
```

Comment:
I try avoid overwriting existing namespaces, maybe something like `:data` would be appropriate?

```
rake data:migrate:student_titles
```

Reply:
Fixed in a13ee0af7704fc8eaa27b8d8702af1fedf76395a